package.json

```diff
@@ -73,6 +73,7 @@
     "get-value": "3.0.1",
     "is": "3.3.0",
     "lodash.clonedeep": "4.5.0",
+    "lodash.difference": "^4.5.0",
```

Comment:
```suggestion
    "lodash.difference": "4.5.0",
```

Reply:
Try to utilise the ramda npm package as lodash alternative.

Reply:
Committed your suggestion